From cde7e5ca775b4d8d2e4f91fbdbe39c0d419733a3 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Wed, 5 Aug 2020 18:02:53 +0800
Subject: [PATCH] Revert "drm/sysfs: add current display mode to sysfs"

This reverts commit 6b6e9cabe90f2d7c164c97de89e25256bf32e73b.
We can get the current mode from cat /d/dri/0/summary for debug.

Change-Id: I13396986a8c996678fb831731063f4d02776b297
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/drm_sysfs.c | 31 -------------------------------
 1 file changed, 31 deletions(-)

diff --git a/drivers/gpu/drm/drm_sysfs.c b/drivers/gpu/drm/drm_sysfs.c
index 6d92f0e93dcb..ecb7b33002bb 100644
--- a/drivers/gpu/drm/drm_sysfs.c
+++ b/drivers/gpu/drm/drm_sysfs.c
@@ -229,47 +229,16 @@ static ssize_t modes_show(struct device *device,
 	return written;
 }
 
-static ssize_t mode_show(struct device *device,
-			   struct device_attribute *attr,
-			   char *buf)
-{
-	struct drm_connector *connector = to_drm_connector(device);
-	struct drm_display_mode *mode;
-	struct drm_crtc_state *crtc_state;
-	bool interlaced;
-	int written = 0;
-
-	if (!connector->state || !connector->state->crtc)
-		return written;
-
-	crtc_state = connector->state->crtc->state;
-	if (!crtc_state)
-		return written;
-
-	mode = &crtc_state->mode;
-
-	interlaced = !!(mode->flags & DRM_MODE_FLAG_INTERLACE);
-	written += snprintf(buf + written, PAGE_SIZE - written,
-			    "%dx%d%s%d\n",
-			    mode->hdisplay, mode->vdisplay,
-			    interlaced ? "i" : "p",
-			    drm_mode_vrefresh(mode));
-
-	return written;
-}
-
 static DEVICE_ATTR_RW(status);
 static DEVICE_ATTR_RO(enabled);
 static DEVICE_ATTR_RO(dpms);
 static DEVICE_ATTR_RO(modes);
-static DEVICE_ATTR_RO(mode);
 
 static struct attribute *connector_dev_attrs[] = {
 	&dev_attr_status.attr,
 	&dev_attr_enabled.attr,
 	&dev_attr_dpms.attr,
 	&dev_attr_modes.attr,
-	&dev_attr_mode.attr,
 	NULL
 };
 
-- 
2.35.3

